config/kube/crds/namespaced/networking.k8s.io_ingresses.yaml

```diff
@@ -0,0 +1,366 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    api-approved.kubernetes.io: https://github.com/kubernetes/enhancements/pull/1111
```

Comment:
Don't we need to _add_ this annotation on the CRDs that already exist in `config/kube/crds` and belong to a relevant API group?

Reply:
Sure, I actually wanted to ask whether we need to do that but then forgot. Let me do it now.

Reply:
Erm, maybe I was mistaken, aren't they already there?

Reply:
I got it, I guessed you mean replacing, and pushing a new commit.

Reply:
Pushed as https://github.com/kubestellar/kubestellar/pull/1367/commits/1707ac866bb56433f978cf387a8edd7519b0bf00.

Reply:
So on triple checking, I see that they were already there but with a URL that references a KCP PR. Yes, better to reference the kube enhancement PR.